### PR TITLE
Update to support new TWAP rules + CFN Template

### PR DESCRIPTION
**Pull request by rcbgr**
Update the application to support the new Prime TWAP rules (i.e., min / hour) and add a sample CloudFormation template to run. Lastly, this plugs in the Prime Go SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +35,77 @@ current price of the asset price on the [Coinbase Exchange](https://exchange.coi
 If the sample application is used to liquidate large positions, there is price action risk that may
 result in trades executing up to 10% lower than the latest price check.
 
+## Usage
+
+### Create Stack
+
+The *Coinbase Prime Liquidator* has a [sample AWS CloudFormation](infra/aws.cfn.yml) (CFN) template that can be deployed to run the application in an Amazon Elastic Container Service (Amazon ECS) cluster. This template creates all of the required resources and can be customized to suit the deployers needs. To deploy the CFN stack, [initialize your AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) and then run:
```
Comment (nmorgan-cb) [inline README.md:42]: Should we add a warning here that these resources will/can cost money if deployed and left idle

File: monitor/liquidator.go [hunk 2]
```diff
@@ -19,43 +19,64 @@ package monitor
...
 
 	l := newLiquidator(config)
 
-	l.monitor()
+	l.stopWaitGroup.Add(1)
+
+	l.running.Store(true)
+
+	go l.monitor()
+
```
Comment (nmorgan-cb) [inline monitor/liquidator.go:56]: should there be logging at this higher level to describe overall state of the liquidator (starting, running, stopped)?

File: util/utils.go [hunk 3]
```diff
@@ -14,28 +14,21 @@
...
-
-	log "github.com/sirupsen/logrus"
+	"crypto/md5"
+	"fmt"
+	"strings"
 )
 
-var primeV1ApiBaseUrl = "https://api.prime.coinbase.com/v1"
+func GenerateUniqueId(params ...string) string {
+	return fmt.Sprintf("%x", md5.Sum([]byte(strings.Join(params, "-"))))
```
Comment (nmorgan-cb) [inline util/utils.go:26]: Why not use the uuid package here?

**Review by nmorgan-cb — CHANGES_REQUESTED** [review]
Just added some comments for clarification asks, but overall LGTM

File: monitor/liquidator.go (hunk 2 above)
Comment (rcbgr) [reply monitor/liquidator.go:56]: There is overall logging in cmd/server/main.go, which I see as a better place to log

File: util/utils.go (hunk 3 above)
Comment (rcbgr) [reply util/utils.go]: I moved the utils to the packages where they are being called and removed the calculate order, which I already moved to the Prime SDK

**Review by cfluke-cb — APPROVED** [review]
small change ask

File: monitor/utils.go [hunk 4]
```diff
@@ -14,28 +14,31 @@
...
+
+	minNotionalPerHour := decimal.NewFromInt(int64(twapMinNotional))
+
+	hours := decimal.NewFromFloat(twapDuration.Hours())
+
+	return value.Div(hours).GreaterThanOrEqual(minNotionalPerHour)
+}
+
+func isFiat(symbol string) (f bool) {
+	if strings.ToLower(symbol) == "usd" {
```
[the remark this answers is not in the document]
Comment (rcbgr) [reply monitor/utils.go:40]: Stablecoins is another configuration, but I'll add euro

File: infra/aws-dev.json [hunk 5]
```diff
@@ -0,0 +1,88 @@
+[
+
```
[the remark this answers is not in the document]
Comment (rcbgr) [reply infra/aws-dev.json:2]: This is the way I've always seen it, but agree with your point and collapsed it down to a line per doc

**Review by nmorgan-cb — APPROVED** [review]

**Review by jc-cb — APPROVED** [review]

**Pull request merged**